packages/twenty-server/src/engine/metadata-modules/object-metadata/object-metadata.service.ts

```diff
@@ -411,6 +413,26 @@ export class ObjectMetadataService extends TypeOrmQueryService<ObjectMetadataEnt
     );
   }
 
+  public async getObjectMetadataStandardIdToIdMap(workspaceId: string) {
```

Comment:
Can you use generateObjectMetadataMaps util in the command instead and remove this method? This should do what you want

Reply:
No this is not the same please look at it more thoroughly I created this type and util because there's a confusion !